test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -52,7 +52,7 @@ def test_tensorflow_train_mnist_cpu(tensorflow_training, ec2_connection, cpu_onl
 
 @pytest.mark.integration("horovod")
 @pytest.mark.model("resnet")
-@pytest.mark.parametrize("ec2_instance_type", TF_EC2_GPU_INSTANCE_TYPE, indirect=True)
+@pytest.mark.parametrize("ec2_instance_type", get_ec2_instance_type("p2.8xlarge", "gpu"), indirect=True)
```

Comment:
Why is this p2.8x? if this needs to persist, then you can leave it this way, or define another constant above if many tests need to be p2.8x in PR context

Reply:
The test was failing for g3.16xlarge, but it turned out that the `shm_large` flag was only enabled for p2.8xlarge instances. When I enabled it for g3.16xlarge as well, it ran successfully.

So this change has been reverted as well.